orca-peering/README.md

```diff
@@ -0,0 +1,139 @@
+# Orca Peering
+
+This is an semi-experimental approach to solve the problem of having multiple `orca` installations (each with its own database) communicate changes with each other, for instance in a multi-region Spinnaker installation or during a database migration.
+
+Note: the word *instance* is used to refer to `orca` below but most likely this is actually a cluster of `orca` instances that all talk to the same database
+ 
+**Definitions:**
+* `peer`
+    An `orca` instance whose database (can be a read replica) we copy data from. Each `orca` instance has an ID, for example `us-east-1` or `us-west-2`.  
```

Comment:
the term "instance" is maybe a bit overloaded. In this case, we mean something like "installation", right?

Reply:
That's why i added the note above... you think it's not enough?

Reply:
Any reason not to just call it a cluster?  In practice, it is essentially a cluster for peering (even if only running a single node).

Reply:
good point, updated